kong/plugins/proxy-cache/schema.lua

```diff
@@ -58,6 +58,11 @@ return {
             default = false,
             required = true,
           }},
+          { cache_lowercase_uri = {
```

Comment:
```suggestion
          { ignore_uri_case = {
```

Or something a bit better since the lower or upper case isn't really relevant here.

Reply:
that sounds good to me, I renamed the variable to `ignore_uri_case` in 8c63957.